styles/globals.css

```diff
@@ -1,3 +1,20 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.accordion-button {
```

Comment:
This file should only have the first 3 lines. Next.js is super picky with CSS and we are going to stick to only Tailwind